store/src/main/scala/geotrellis/store/cog/package.scala

```diff
@@ -32,13 +32,8 @@ package object cog extends Implicits {
     }
   }
 
-  implicit class withExtentMethods(extent: Extent) {
+  implicit class ExtentLayoutOps(extent: Extent) {
```

Comment:
should avoid changing class names for little reason, saves a binary comparability bug for somebody out there in the universe.